sympy/physics/wigner.py

```diff
@@ -216,8 +216,14 @@ def wigner_3j(j_1, j_2, j_3, m_1, m_2, m_3):
     - Jens Rasch (2009-03-24): initial version
     """
 
-    j_1, j_2, j_3, m_1, m_2, m_3 = map(_int_or_halfint,
-                                       [j_1, j_2, j_3, m_1, m_2, m_3])
+    j_1 = sympify(j_1)
+    j_2 = sympify(j_2)
+    j_3 = sympify(j_3)
+    m_1 = sympify(m_1)
+    m_2 = sympify(m_2)
+    m_3 = sympify(m_3)
+
+    j_1, j_2, j_3, m_1, m_2, m_3 = map(_int_or_halfint, [j_1, j_2, j_3, m_1, m_2, m_3])
```

Comment:
You can delete all the previous lines that were added and write
```python
j_1, j_2, j_3, m_1, m_2, m_3 = \
    map(_int_or_halfint, map(sympify,
        [j_1, j_2, j_3, m_1, m_2, m_3]))
```

Reply:
Alright, I'll do it by EOD! Thank you for the review.